spec.bs

```diff
@@ -3990,6 +4007,55 @@ at the expense of some utility.
   </wpt>
 </div>
 
+<h3 id=credentialless-monkeypatch>Iframe credentialless</h3>
+
+The [[!IFRAME-CREDENTIALLESS]] specification defines a new object, the [=page credentialless 
+nonce=]. At a high level, the [=fenced frame config instance/partition nonce=] serves the same
+purpose as the [=page credentialless nonce=] (partitioning storage and network), but scoped to each
+fenced frame rather than to each credentialless iframe. The [=fenced frame config instance/
```

Comment:
Is page credentialless nonce really scoped to each credentialless iframe? The definition you link to seems to be tied to a top-level browsing context, which spans many iframes.